<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ajoutcommande</class>
 <widget class="QMainWindow" name="ajoutcommande">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>35</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ajouter une nouvelle commande</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>801</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Veuillez remplir les champs suivants :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>231</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mail du client :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>231</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Taille de casier nécessaire : </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>231</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Poids du colis (kg) :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>231</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nom du commerçant :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>231</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Adresse du commerçant :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_mail">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>180</y>
      <width>431</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>ishakabdoul@gmail.com</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_poids">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>431</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nom">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>360</y>
      <width>431</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_adresse">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>420</y>
      <width>431</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_taille">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>240</y>
      <width>431</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ajouter">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>480</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_annuler">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>480</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Annuler</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
